NLibrary/Classes/PlaylistMVVM/PlaylistModel.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+public struct PlaylistModel {
+    public var playlistId: String = ""
+    public var playlistName: String = ""
+    public var producerEmail: String = ""
+    public var producerId: String = ""
+    public var numSongs: Int = 0
+    public var basicSongItems: [BasicSongItem] = []
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5415919)